src/modules/conversation-handler/conversationGpt.ts

```diff
@@ -44,7 +56,23 @@ export async function conversationGpt(
         if (response) {
           chat.push({ content: response.completion, role: "system" });
           usage += response.usage;
-          ctx.reply(response.completion!);
+          price = response.price;
+          totalPrice += price
+          console.log(usage, totalPrice);
```

Comment:
If you need this logs, better to use logger

Reply:
Deleted. I was using the console.log as code flags.